src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -46,12 +46,26 @@ export default class CourtOrderPoa extends Vue {
   @Prop({ default: false })
   private validate
 
+  /** Draft court order number. */
+  @Prop({ default: '' })
+  private draftCourtOrderNumber
+
+  /** Draft plan of arrangement. */
+  @Prop({ default: false })
+  private draftPlanOfArrangement
```

Comment:
I find this variable name a teensy bit confusing.... perhaps isDraftPlanOfArrangement ?

Reply:
definitely agree! Will update this

Reply:
updated 👍